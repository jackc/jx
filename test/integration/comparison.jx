puts("0 == 0 ")
if 0 == 0
  puts("success")
else
  puts("failure")
end
puts("\n")

puts("0 != 1 ")
if 0 != 1
  puts("success")
else
  puts("failure")
end
puts("\n")

puts("0 < 1 ")
if 0 < 1
  puts("success")
else
  puts("failure")
end
puts("\n")

puts("1 > 0 ")
if 1 > 0
  puts("success")
else
  puts("failure")
end
puts("\n")

puts("1 <= 1 ")
if 1 <= 1
  puts("success")
else
  puts("failure")
end
puts("\n")

puts("1 >= 1 ")
if 1 >= 1
  puts("success")
else
  puts("failure")
end
puts("\n")
