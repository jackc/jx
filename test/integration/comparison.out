0 == 0 success
0 < 1 success
1 > 0 success
1 <= 1 success
1 >= 1 success
